### PR TITLE
aria2: fix some typo and init script mistakes

### PR DESCRIPTION
**Pull request by sotux**
1. fix some typo in aria2.init script
2. fix log option according to the luci-app-aria2 setting

### CODE REVIEW

File: net/aria2/files/aria2.init [hunk 1]
```diff
@@ -323,8 +323,7 @@ aria2_start() {
 	fi
 
 	procd_open_instance "${NAME}.${section}"
-	procd_set_param command "$PROG"
-	procd_append_param command --conf-path="${config_file}"
+	procd_set_param command $PROG --conf-path=${config_file}
```
Comment (hnyman) [inline net/aria2/files/aria2.init]: why this change?

Comment (sotux) [reply net/aria2/files/aria2.init]: I think using procd_set_param is enough.

File: net/aria2/files/aria2.init [hunk 2]
```diff
@@ -188,10 +188,10 @@ aria2_start() {
 	append_setting "input-file=${session_file}"
 	append_setting "save-session=${session_file}"
 
-	if [ -z "$enable_logging" ]; then
+	if [ -z "$enable_log" ]; then
```
Comment (yousong) [inline net/aria2/files/aria2.init]: this option rename will break existing configs, causing unneeded surprises.  please do not do it.

Comment (hnyman) [reply net/aria2/files/aria2.init]: @sotux 

> fix log option according to the luci-app-aria2 setting

If your reasoning was the LuCI app has a different options than aria itself, then you should fix the LuCI app, not the main package.

Comment (sotux) [reply net/aria2/files/aria2.init]: OK. Then I will fix only typo in this package and fix the log option in the luci app. Thanks.
